components/PokemonListItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useMemo } from "react";
+import Link from "next/link";
+import styles from "./pokemonListItem.module.css";
+import Image from "next/image";
+
+type PokemonListItemProps = {
+  pokemon: Record<string, any>;
```

Comment:
Let's create a type
`types/pokemon.ts`
That should help us make it reusable across a lot of places

Reply:
Also no `any` please

Reply:
Done as stated